includes/classes/blocks/class-template-parts.php

```
@@ -206,24 +206,47 @@ protected function maybe_create_template_part( $slug, $args, $theme ) {
+				return $post && $post->post_name === $slug;
+			}
+		);
+
+		if ( defined( 'WP_DEBUG' ) && WP_DEBUG ) {
+			error_log( sprintf( 'Template Parts: Checking %s for theme %s', $slug, $theme ) );
+			error_log( sprintf( 'Template Parts: SQL Query: %s', $query->request ) );
+			error_log( sprintf( 'Template Parts: Found %d template parts for theme %s', count( $query->posts ), $theme ) );
+			error_log( sprintf( 'Template Parts: Found %d matching slug %s', count( $existing ), $slug ) );
+			if ( ! empty( $existing ) ) {
+				// Check what theme is actually assigned
+				$first_id = reset( $existing );
```

PHPMD (2.15.0)

The variable $first_id is not named in camelCase. (undefined)

(CamelCaseVariableName)